hcipy/aperture/generic.py

```diff
@@ -363,21 +364,21 @@ def make_rotated_aperture(aperture, angle):
 	return lambda grid: Field(aperture(grid.rotated(-angle)), grid)
 
 def make_shifted_aperture(aperture, shift):
-	'''Create a rotated version of `aperture`.
+	'''Create a shifted version of `aperture`.
 
 	Parameters
 	----------
 	aperture : Field generator
 		The aperture to rotate.
-	angle : scalar
-		The angle in radians by which to rotate the aperture.
+	shift : array_like
+		The shift of the aperture.
 
 	Returns
 	-------
 	Field generator
-		The rotated aperture.
+		The shifted aperture.
 	'''
-	return lambda grid: Field(aperture(grid.shifted(shift)), grid)
+	return lambda grid: Field(aperture(grid.shifted([-shift[0], -shift[1]])), grid)
```

Comment:
Good catch on the minus sign. I didn't see that during my review. But can you put `-np.array(shift)` instead, so this works for higher dimensional shifts as well. (Not that we'd ever use that, but it's cleaner code.)